RELEASENOTES.md

```diff
@@ -6,6 +6,8 @@ Releases, starting with 9/2/2021, are listed with the most recent release at the
 __Bug Fixes__:
 
 #1426 Sequential.eval() does not put model into eval mode<br/>
+#1440 `torch.optim.lr_scheduler.PolynomialLR` `power` type has been corrected, is now double.<br/>
```

Comment:
Hey, just like in the other PR, the number should point to an issue or discussion thread and since this is not one, I say we follow the pattern from here: https://github.com/dotnet/TorchSharp/blob/5ba14022749a20b98798105a32988da59f2356ca/RELEASENOTES.md?plain=1#L287

Reply:
thanks, removed the number.
feel free to mark resolved if ok.